candle-examples/examples/llava/readme.md

```diff
@@ -0,0 +1,108 @@
+# candle-llava
+
+implement LLaVA using candle
+
+This example is from [candle-llava](https://github.com/chenwanqq/candle-llava)
+
+The code is based on [https://github.com/haotian-liu/LLaVA](https://github.com/haotian-liu/LLaVA), Hence the llava-hf version of config may perform differently.
+
+
+The llava-hf models contain tokenizer.json, so if you want pure-rust experience, I suggest you to use llava-hf version.
+
+## model zoo
+* [liuhaotian/LLaVA](https://huggingface.co/liuhaotian)
+* [llava-hf](https://huggingface.co/llava-hf)
+
+Right now I have tested on liuhaotian/llava-v1.6-vicuna-7b and llava-hf/llava-v1.6-vicuna-7b-hf. The memory use might have room for optimization.
+
+
+## eval
+
+### single-image
+```bash
+ cargo run --example llava --features cuda  # default args, use  llava-hf/llava-v1.6-vicuna-7b-hf, default-image is llava_logo.png, prompt is "is this a cat?"
+```
+
+## task
```

Comment:
Probably not meaningful to have the task list here? You may just want to mention some of the unsupported bits if they are important.